CMake/include_os.cmake

```diff
@@ -7,5 +7,6 @@
     endif()
 
     if(ANDROID_NDK_TOOLCHAIN_INCLUDED)
+        set(ANDROID_USB_HOST_UVC ON)
```

Comment:
Considering this is the only way to get the Android backend to work it will have to be via usbhost. I will see if I can remove the variables in CMake.

Reply:
Removed in 2b3972ed2f3b192180ec9bcb5d6af268daa51fc8